### PR TITLE
Update applications.py

### PR DESCRIPTION
**Pull request by tsv19su254052**
corrected indents and hyphens

### CODE REVIEW

File: fastapi/applications.py [hunk 1]
```diff
@@ -196,18 +185,16 @@ def build_middleware_stack(self) -> ASGIApp:
...
-                openapi_version=self.openapi_version,
-                description=self.description,
-                terms_of_service=self.terms_of_service,
-                contact=self.contact,
-                license_info=self.license_info,
-                routes=self.routes,
-                tags=self.openapi_tags,
-                servers=self.servers,
-            )
+            self.openapi_schema = get_openapi(title=self.title,
```
Comment (JarroVGIT) [inline fastapi/applications.py]: I have left a remark here: https://github.com/tiangolo/fastapi/pull/5016#issuecomment-1152278151

**Review by JarroVGIT — CHANGES_REQUESTED** [review]
Recommendation: retract as it makes the code not PEP-8 compliant. See https://github.com/tiangolo/fastapi/pull/5016#issuecomment-1152278151

**Comment by tsv19su254052** [comment]
OK. It is a C++ Style.
I just wanted to show my style. If these changes do not comply with the PEP-8 standard, then you can refuse them

**Review by MarcusVinniciuss — APPROVED** [review]

**Comment by tiangolo** [comment]
Thanks for the interest! FastAPI uses Black for formatting, and pre-commit to enforce it. In fact, it fixes the format and commits on top of PRs in CI, that's why you see a commit on top of yours and why it seems the PR has no changes.

As format is already handled, I'll close this PR now. But thanks for the interest! :coffee:

**Pull request closed without merging**